test-tests/shared/src/main/scala/zio/test/TestUtils.scala

```diff
@@ -26,14 +23,25 @@ object TestUtils {
       results.forall { case Spec.TestCase(_, test) => test.map(f); case _ => ZIO.succeed(true) }
     }
 
-  final def ignored[L, E, S](spec: ZSpec[environment.TestEnvironment, E, L, S]): ZIO[Any, Nothing, Boolean] = {
+  final def isFailed[L, E, S](spec: ZSpec[environment.TestEnvironment, E, L, S]): ZIO[Any, Nothing, Boolean] =
```

Comment:
Was that also conflicting?

Maybe `failedWith` should be renamed too?

Reply:
These combinators are only used in the internal test suite for ZIO Test. Once I renamed `ignore` to `ignored` I created another conflict with an internal combinator used here so I changed that to `isIgnored` and then just changed the other internal combinators for consistency. Yes I think we could rename `failedWith` too.  Let me do that now.

Reply:
Actually realized several of those combinators are not being used anymore now that the tests that relied on them have been migrated to ZIO Test so was able to remove them entirely.